scripts/cli/start.ts

```diff
@@ -0,0 +1,32 @@
+const concurrently = require('concurrently');
```

Comment:
can this file be in ts? looks like we are already using ts-node

Reply:
This is ts file. Are you referring to `require`?